Description: customise AP network credentials for pi-topOS.
Forwarded: not-needed
Author: pi-top <deb-maintainers@pi-top.com>
Last-Update: 2021-07-14

--- a/wifi-ap-sta
+++ b/wifi-ap-sta
@@ -16,7 +16,8 @@

 initialise_ap_values() {
	AP_SSID="${AP_SSID:-$(produce_ssid)}"
	AP_PASSWORD="${AP_PASSWORD:-$(produce_password)}"
-	AP_IFACE="${AP_IFACE:-ap0}"
+	AP_IFACE="${AP_IFACE:-wlan_ap0}"
 	AP_MAC="${AP_MAC:-99:88:77:66:55:44}"
 	WIFI_IFACE="${WIFI_IFACE:-wlan0}"
 	STATIC_IP_PREFIX="${STATIC_IP_PREFIX:-192.168.90}"
