Description: customise AP network credentials for pi-topOS.
Forwarded: not-needed
Author: pi-top <deb-maintainers@pi-top.com>
Last-Update: 2021-07-14

--- a/wifi-ap-sta
+++ b/wifi-ap-sta
@@ -16,7 +16,8 @@

 initialise_ap_values() {
 	AP_SSID="${AP_SSID:-$(hostname)}"
-	AP_IFACE="${AP_IFACE:-ap0}"
+	AP_PASSWORD="pi-top1234"
+	AP_IFACE="${AP_IFACE:-wlan_ap0}"
 	AP_MAC="${AP_MAC:-99:88:77:66:55:44}"
 	WIFI_IFACE="${WIFI_IFACE:-wlan0}"
 	WIFI_IFACE_MAC="$(cat /sys/class/net/"${WIFI_IFACE}"/address)"
